OneCryptoPkg/OneCryptoLoaders/OneCryptoLoaderStandaloneMm.inf

```diff
@@ -0,0 +1,52 @@
+## @file
```

Comment:
I don't think you need two files here because they are the same. Just the integrate needs them, in your dsc you would just swap the entry point lib

Reply:
I think I might leave that for follow up task and think more about it.

Mostly I don't know if I want a define because that feels too much like "flavors" unless there's some DSC magic I'm unfamiliar with. The organization as they are - are tightly defined which is a property that I like. And there is a matching integration file which does need to exist on the other side.